Automattic/wp-calypso
# New domain mapping status screen

#### Changes proposed in this Pull Request

* Introduce the new domain mapping status screen

#### Testing instructions

* open domain mapping with/without plan
* check the expiration dates
* check domain mapping without proper nameservers
* check subdomain mapping without proper CNAME/NS records
* check that "Renew your plan" is shown when mapping is bundled with plan

Here's how it looks:

Active mapping:
<img width="740" alt="Screenshot 2020-03-03 at 16 13 02" src="https://user-images.githubusercontent.com/1355045/75784584-185f3d00-5d6b-11ea-80d4-6758be2406f8.png">

Active with plan:
<img width="747" alt="Screenshot 2020-03-03 at 16 16 51" src="https://user-images.githubusercontent.com/1355045/75784636-3167ee00-5d6b-11ea-9210-5b961ab4a08f.png">

Expiring soon:
<img width="742" alt="Screenshot 2020-03-03 at 16 10 23" src="https://user-images.githubusercontent.com/1355045/75784601-201ee180-5d6b-11ea-8f5f-5653b4458574.png">

Domain not pointing to us:
<img width="739" alt="Screenshot 2020-03-03 at 16 20 17" src="https://user-images.githubusercontent.com/1355045/75784710-49d80880-5d6b-11ea-8f75-dd12df2223a8.png">

Subdomain not pointing to us:
<img width="741" alt="Screenshot 2020-03-03 at 16 19 31" src="https://user-images.githubusercontent.com/1355045/75784726-50ff1680-5d6b-11ea-99b4-c28357512e92.png">

depends on: D39734-code
